packages/fxa-content-server/app/scripts/templates/pair/failure.mustache

```diff
@@ -10,7 +10,7 @@
     <div class="success"></div>
 
     <form novalidate>
-      <div class="graphic graphic-pair-failure">{{#t}}Pairing not successful{{/t}}</div>
+      <div class="bg-image-pair-fail"></div>
```

Comment:
I mention this in my original comment but calling out here, I'll be filing a follow up. I don't think it's useful to have this text read when it's exactly the same text as they're read on the previous element (the header) so I nixed those but want to see what's most ideal.

Also changed the QR code one to read "QR code". That seems useful.

> Evaluate aria-labels/alt text for images - I removed some in this PR that simply repeated the header text which is unhelpful and left others, but would be nice to go through them at some point and decide what should be read by screenreaders since some images could be considered decorative